package.json

```diff
@@ -2,7 +2,7 @@
   "name": "websockets-vanilla",
   "version": "1.0.0",
   "engines": {
-    "node": "6.2.0"
+    "node": "13.1.x"
```

Comment:
I tried to stick with the LTS major versions (`10.x`, `12.x`) unless there is some feature from a current version (`13.x`) that's necessary